server/routes/barcode/contacts/CreateContactController.ts

```diff
@@ -50,6 +50,8 @@ export default class CreateContactController {
     try {
       const prisonAddress = await this.prisonRegisterService.getPrisonAddress(newRecipient.prisonId)
       this.addRecipient(req, newRecipient, prisonAddress)
+      req.session.findRecipientForm = undefined
```

Comment:
After creating each recipient, clear the forms, which allows us to present empty forms to the user when adding more recipients
It also meant I could remove the clearing of the forms in GenerateBarcodeImageController